CHANGELOG.md

```diff
@@ -29,9 +102,9 @@ By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/p
 
 ## 🐛 Fixes
 
-### Prevent span attributes from being formatted to write logs 
+### Prevent span attributes from being formatted to write logs
```

Comment:
This looks a bit off.

Reply:
Fwiw, I fixed the bug that introduced the trailing space in the first place in https://github.com/apollographql/router/pull/3152.  So hopefully less of this in the future.